dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -129,7 +129,9 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
 
     inline wil::com_ptr<INotificationsLongRunningPlatform> GetNotificationPlatform()
     {
-        return wil::CoCreateInstance<NotificationsLongRunningPlatform, INotificationsLongRunningPlatform>(CLSCTX_LOCAL_SERVER);
+        wil::com_ptr<INotificationsLongRunningPlatform> platform;
+        THROW_IF_FAILED(CoCreateInstance(CLSID_NotificationsLongRunningPlatform, nullptr, (CLSCTX_LOCAL_SERVER | CLSCTX_ALLOW_LOWER_TRUST_REGISTRATION), IID_PPV_ARGS(&platform)));
```

Comment:
If you prefer WIL, there is a wil::CoCreateInstance that takes CLSID as a normal parameter (rather than the "CLSID carried by the UUID attribute of a template parameter" form you previously used); you don't need to switch to the raw API.